priv/app/graph.jsx

```diff
@@ -47,6 +41,35 @@ export default class Graph extends React.Component {
       errorMsg = <strong>  -  communication error</strong>;
     }
 
+    const data = {
+      x: "x",
+      columns: this.state.columns,
+      axes: {
+        count: "y2"
+      },
+      colors: {
+        count: "#98fb98",
+        max: "#8c2a04",
```

Comment:
Let's use `"` everywhere here then.
And please use uppercase letters for colors. 😉